app/controllers/api/v1/events_controller.rb

```diff
@@ -92,29 +90,6 @@ def update(params)
       else
         render json: { errors: event.errors }
       end
-
-    end
-
-    def is_user_attending?(user, event)
-      event.users.include?(user)
-    end
-
-    def user_has_already_rated(event, user)
-      event.ratings.exists?(user_id: user.id)
-    end
-
-    def remove_rating(event, user)
-      event.ratings.destroy(event.ratings.where(:user_id => user.id))
-    end
-
-    def find_rating(event, user)
```

Comment:
honestly I not sure that my implmentation is any better than `remove_rating(event, user) if user_has_already_rated(event, user)`

its moved to `rating = Rating.find_or_initialize_by(user_id: user.id, event_id: event.id)`